_data/patcher-landing-page-tabs-dependencies-list.yml

```diff
@@ -0,0 +1,7 @@
+- id: cli
+  title: Use the CLI
+  content: /assets/img/landing-page-patcher/patcher-cli.gif
```

Comment:
I think using a tool like https://asciinema.org/ will allow you to have a purely text based recording of the terminal, rather than an image that is a tiny bit blurry and hard to read. This can be done in a follow-up issue / PR.

Reply:
Having this image for now would be better?

![carbon](https://user-images.githubusercontent.com/9384479/129921996-4c74a953-a929-4275-8f93-9dc29e2aff75.png)